vllm/model_executor/layers/fused_moe/config.py

```diff
@@ -51,14 +53,18 @@ def get_config_quant_dtype(
     use_int8_w8a8: bool,
     use_int8_w8a16: bool,
     use_int4_w4a16: bool,
-    use_mxfp4_w4a4: bool,
+    ocp_mx_scheme: Optional[str],
 ) -> Union[None, torch.dtype, str]:
     if use_fp8_w8a8:
         return torch.float8_e4m3fn
     elif use_int8_w8a8:
         return torch.int8
-    elif use_mxfp4_w4a4:
-        return "mxfp4"
+    elif ocp_mx_scheme == "w_fp4_a_fp4":
```

Comment:
Nit: Instead of doing the string comparisons can you compare with the `OCP_MX_Scheme` enum that you added?